pkg/logger/common/common.go

```diff
@@ -65,7 +66,7 @@ func init() {
 		}
 		FullAccessLogMaxBodySize = int(value)
```

Comment:
what will happen if the parsing step got failed and FullAccessLogMaxBodySize got 0 value?